demo-app/package.json

```diff
@@ -59,11 +53,9 @@
     "ember-qunit": "^8.1.0",
```

Comment:
There's probably some other stuff we can remove like qunit, but I don't think trimming down the dependencies is important enough to block this PR.